### DISCUSSION TITLE
Wix4 - Any way to tell wixtoolset not to build (ignore) a file

### DISCUSSION THREAD

**Discussion by wmanning** [wixtoolset/issues#7366] [Questions] [answered]
In a C# project for instance you can use following to tell the .net compiler to ignore a file

```xml 
<ItemGroup>
    <Compile Remove="somefile.cs" />
</ItemGroup>
```

Any equivalent in Wix?

There's some wix files I want to store in the project directory as informational only.

**Comment by wmanning** [comment]
Actually you can do:

```
<ItemGroup>
    <EmbeddedResource Remove="somefile..." />
  </ItemGroup>
```

**Reply by wmanning** [reply]
I was wrong.  Didn't work.

**Comment by robmen** [comment, marked as answer]
Depends on the file type, the following should work:

```xml
<ItemGroup>
    <Compile Remove="SomeCodeFile.wxs" />
    <EmbeddedResource Remove="SomeLocFile.wxl" />
</ItemGroup>
```

Or you can [turn off the automatic globbing](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#default-item-inclusion-properties).

**Reply by wmanning** [reply]
Tried both Compile and EmbeddedResourse but didn't work.  The DefaultItemExcluded in your link worked:
```xml
<PropertyGroup>
    <DefaultItemExcludes>$(DefaultItemExcludes);Theme\HyperlinkTheme Originals\**/*</DefaultItemExcludes>
</PropertyGroup>
```

**Comment by wmanning** [comment]
Didn't work (tried both Compile and EmbeddedResource just in case).  Had a wxl file I wanted to ignore as it's informational only and was causing duplicate symbols errors (think that was the error).

But DefualtItemExcludes in your link worked:

'''xml
<PropertyGroup>
    <DefaultItemExcludes>$(DefaultItemExcludes);Theme\HyperlinkTheme Originals\**/*</DefaultItemExcludes>
</PropertyGroup>
''''
